vulyk/blueprints/gamification/__init__.py

```diff
@@ -20,12 +20,40 @@
 
 from . import listeners
 from .models.events import EventModel
+from .. import VulykModule
 
 __all__ = [
     'gamification'
 ]
 
-gamification = flask.Blueprint('gamification', __name__)
+
+class GamificationModule(VulykModule):
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+
+        self.config["levels"] = {
+            k: 1 if k == 1 else (k - 1) * 25 for k in range(1, 51)
+        }
+
+    def get_level(self, points):
+        """
+        Obtains the level that corresponds to a number of points
+
+        :param points: Number of points
+        :type points: Decimal
+
+        :return: Level
+        :rtype: int
+        """
+
+        for k in sorted(self.config["levels"].keys(), reverse=True):
```

Comment:
To prevent redundant sorting and iteration for every single time the method is being called, we may:
- use OrderedDict;
- swap keys and values to reduce values dereferencing;
- having hardcoded levels above, we may just use floor division by 25.